applications/zigbee_weather_station/README.rst

```diff
@@ -133,8 +133,14 @@ Power switch (SW1):
     Used for switching Thingy:53 on and off.
 
 Button (SW3):
-    Used to start and cancel the Identify mode.
-    Additionally, pressing the button indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was stopped (for example, after timeout).
+    Depending on how long the button is pressed:
+    
+    * If pressed for less than five seconds, it starts or cancels the Identify mode.
+    * If pressed for five seconds, it initiates the factory reset of the device.
+      The length of the button press can be edited using the :kconfig:`CONFIG_FACTORY_RESET_PRESS_TIME_SECOND` Kconfig option from :ref:`lib_zigbee_application_utilities`.
+      Releasing the button within this time cancels the factory reset procedure.
+    
+    Additionally, pressing the button shortly or releasing it before five second has elapsed indicates user input, which allows a Sleepy End Device to start the network rejoin procedure if it was previously stopped (for example, after timeout).
```

Comment:
I don't know what is incorrect or how would you like to change it.

Reply:
@sebastiandraus: based on chat in Teams - I'll split the behaviors: identification and user input indication will be done only when the button is released (within 5s time window, otherwise a reset is done and button release has no effect)